client/src/modules/vouchers/toolkit/support_patient.js

```diff
@@ -0,0 +1,189 @@
+angular.module('bhima.controllers')
+.controller('SupportPatientKitController', SupportPatientKitController);
+
+// DI definition
+SupportPatientKitController.$inject = [
+  '$uibModalInstance', 'NotifyService', 'CashboxService', 'SessionService', 'data', 'bhConstants',
+  'DebtorService', 'PatientInvoiceService'
+];
+
+// Import transaction rows for a Support Patient
+function SupportPatientKitController(Instance, Notify, Cashbox, Session, Data, bhConstants, Debtors, Invoices) {
+  var vm = this;
+
+  // global variables
+  vm.enterprise = Session.enterprise;
+  vm.gridOptions = {};
+  vm.tool = Data;
+  vm.patientInvoice = false;
+
+  var MAX_DECIMAL_PRECISION = bhConstants = bhConstants.precision.MAX_DECIMAL_PRECISION;
+
+  // expose to the view
+  vm.selectPatientInvoices = selectPatientInvoices;
+  vm.close = Instance.close;
+  vm.import = submit;
+  vm.loadInvoice = loadInvoice;
+  vm.onSelectAccount = onSelectAccount;
+
+  // debtors from store
+  Debtors.store()
+  .then(function (data) {
+    vm.debtors = data;
+  })
+  .catch(Notify.handleError);
+
+  //  optimization with `Store` will be well
+  Invoices.read()
+  .then(function (data) {
+    vm.invoices = data;
+  })
+  .catch(Notify.handleError);
+
+  // get debtor group invoices
+  function selectPatientInvoices(debtorId) {
+    // load debtor invoices
+    vm.debtorUuid =  debtorId;
+
+    Debtors.invoices(debtorId, { balanced : 0 })
+      .then(function (invoices) {
+        vm.gridOptions.data = invoices || [];
+
+        // total amount
+        vm.totalInvoices = vm.gridOptions.data.reduce(function (current, previous) {
+          return current + previous.balance;
+        }, 0);
+
+        // make sure we are always within precision
+        vm.totalInvoices = Number.parseFloat(vm.totalInvoices.toFixed(MAX_DECIMAL_PRECISION));
+        
+      })
+      .catch(Notify.handleError);
+  }
+
+  function loadInvoice(debtor){
```

Comment:
_Technically_, this is the `patient` not the `debtor`.  It is confusing to assign the `patient` to `vm.debtor`.  Could you use `vm.patient` as the variable instead?